supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc

```diff
@@ -45,6 +45,18 @@
 
 *See also*:
 
+[discrete]
```

Comment:
Would it be a good idea to provide a link to the official Technology Preview page that is available on the RHCP?

https://access.redhat.com/support/offerings/techpreview